jsonnet/telemeter/metrics.jsonnet

```diff
@@ -46,4 +46,16 @@
   // cluster alerting for the purposes of preventing upgrades from
   // failing for end users.
   '{__name__="ALERTS",alertstate="firing"}',
+  // node_cpu_utilisation is used to report the current avg use of CPU
+  // in the cloud.openshift.com portal.
+  'sum({__name__="node:node_cpu_utilisation:avg1m"})',
+  // node_memory_utilisation is used to report the current use of
+  // memory in the cloud.openshift.com portal.
+  'sum({__name__="node:node_memory_utilisation:ratio"})',
+  // node_filesystem_free_bytes is used to
```

Comment:
This comment is referencing the wrong metric. It should say `node_filesystem_size_bytes`

Reply:
Sorry, fixing.